go.mod

```diff
@@ -13,19 +13,19 @@ require (
 	github.com/lightninglabs/aperture v0.1.18-beta
 	github.com/lightninglabs/faraday v0.2.8-alpha.0.20220624141723-ddd3cd123e62
 	github.com/lightninglabs/lightning-node-connect v0.1.12-alpha
-	github.com/lightninglabs/lndclient v0.15.0-10
-	github.com/lightninglabs/loop v0.19.1-beta.0.20220623090540-08209f61e304
-	github.com/lightninglabs/loop/swapserverrpc v1.0.1
+	github.com/lightninglabs/lndclient v0.15.1-1
```

Comment:
Should use `v0.15.1-5` which allows you to remove the `.rc1` suffix from the `lnd` version (without go mod changing it back).